Hestia/Hestia/backend/speech_recognition/SpeechRecognition.cs

```diff
@@ -78,7 +78,7 @@ public void StartRecording(out int warningStatus)
                 if (err != null)
                 {
                     Console.WriteLine(strings.speechRecordError);
-                    viewController.ProcessSpeech(null);
```

Comment:
In `ProcessSpeech` we check if the result is null, because this is done after calling `result.Contains(...)` you got the null exception. So what we should do is still pass null to `ProcessSpeech` and check if the result is null before doing other string comparisons.

Reply:
see 052565fa6f6feb0218419dfe41b8f3ed0c8c9960